docs/content.zh/docs/dev/table/sql/queries/window-agg.md

```diff
@@ -40,7 +40,9 @@ Unlike other aggregations on continuous tables, window aggregation do not emit i
 
 ### Windowing TVFs
 
-Flink supports `TUMBLE`, `HOP` and `CUMULATE` types of window aggregations, which can be defined on either [event or processing time attributes]({{< ref "docs/dev/table/concepts/time_attributes" >}}). See [Windowing TVF]({{< ref "docs/dev/table/sql/queries/window-tvf" >}}) for more windowing functions information.
+Flink supports `TUMBLE`, `HOP` and `CUMULATE` types of window aggregations.
+For SQL queries on streaming tables, the time attribute field of a window table-valued function must be on either [event or processing time attributes]({{< ref "docs/dev/table/concepts/time_attributes" >}}). See [Windowing TVF]({{< ref "docs/dev/table/sql/queries/window-tvf" >}}) for more windowing functions information.
+For SQL on batch tables, the time attribute field of a window table-valued function must be an attribute of type `TIMESTAMP` or `TIMESTAMP_LTZ`. 
```

Comment:
For batch queries

Reply:
I'm not sure this update is good because it's weird for user to distinguish batch queries or streaming queries. 
How about 'For SQL mode'?

Reply:
+1 for streaming/batch mode